### PR TITLE
Code style updates for diffParse

### PR DESCRIPTION
**Pull request by darkwing**
This file was structured differently than the other modules, so I updated it:

1.  No `module.exports`, just used `exports`
2.  The function was defined as a var, moved it to a named function
3.  Didn't like `work` as a var name
4.  Not computing the lines numbering on each iteration.

### CODE REVIEW

**Comment by openjck** [comment]
Awesome! :fireworks:

File: diffParse.js [hunk 1]
```diff
@@ -25,4 +27,4 @@ var lineToIndex = function(diff, lineNumber) {
     return -1;
 };
```
Comment (openjck) [inline diffParse.js]: We can remove the semicolon now that it's not a `var`.

**Comment by darkwing** [comment]
Updated!

**Comment by openjck** [comment]
Nice updates! :100:

**Pull request merged**